Lib/multiprocessing/pool.py

```diff
@@ -587,12 +597,18 @@ def _handle_results(outqueue, get, cache):
                 util.debug('result handler got sentinel')
                 break
 
-            job, i, obj = task
-            try:
-                cache[job]._set(i, obj)
-            except KeyError:
-                pass
-            task = job = obj = None
+            job, i, (task_info, value) = task
```

Comment:
We know that `value` is always `pid` here, right? Can we just call it that? (I spent a minute or two trying to figure out whether `value` is hashable and eventually traced it back to the source...)

Reply:
Similarly, `task_info` is really the result, yes? Or `task_is_complete`?

Reply:
Obviously I'm not very familiar with this module, but given you've got such wonderfully descriptive names everywhere else in the PR, these ones stand out as a bit vague :)

Reply:
When `task_info` is `None`, then `value` will be the pid but if `task_info` is `True` then `value` will be the result of the task or if `False` then `value` will be an exception object.  I'm totally agreed that `value` is vague but I struggled to come up with a more appropriate name.  I do take some small consolation that `(task_info, value)` is at least a little more helpful than `obj`. :)